scheduler/src/cook/mesos/util.clj

```diff
@@ -89,6 +89,16 @@
                 x))
             m))
 
+(defn entity->map
```

Comment:
I mean an explicit test where an entity with nested components/entities get loaded. Which unit test covers this?